lib/cisco_node_utils/cmd_ref/overlay_global.yaml

```diff
@@ -3,7 +3,7 @@
 _exclude: [ios_xr]
 
 anycast_gateway_mac:
-  _exclude: [N3k]
+  _exclude: [N3k, N7k]
```

Comment:
I think that would be prudent. Could probably just add a check here to do the needful:

https://github.com/cisco/cisco-network-node-utils/blob/develop/tests/test_overlay_global.rb#L94

...which would do:

```
  skip('Test requires Multi-Tenancy Full compatible linecard on this platform') if
    node.product_id[/N7/] && !mt_full_interface?
```